test/compression.js

```diff
@@ -304,6 +312,26 @@ describe('compression()', function () {
     .expect(200, done)
   })
 
+  if (http2) {
+    it('should work with http2 server', function (done) {
```

Comment:
I would put all the http2 tests within their own describe block and if `http2` is not there, have it execute `describe.skip` instead of `describe` so mocha still sees the same number of tests in all Node.js versions 👍

Reply:
Do you mean something like this?

```
var describeIfHttp2 = http2 ? describe : describe.skip;
describeIfHttp2('http2', function () {
  it('should work', function () {
    ...
  })
})
```

Reply:
Yep! I thought there was an example in this repo, but I was mistaken. Here's an example you can check out if you needed to (though you're correct): https://github.com/pillarjs/finalhandler/blob/df647b084963b7666061966f38501f041d29a094/test/test.js#L16

Reply:
Done